docs/release/readme.md

```diff
@@ -34,12 +40,21 @@ On creating a release it's exactly the same as above except the tag will be the
 production -> ../tf-autobuilder/v3.4.5
 ```
 
-> NOTE: during the writing of this docs, not all networks are using this release pipeline and they might still using the old style. Hopefully this will phaseout as soon as possible to use the procedure described in this document.
-
 ## Creating the links
 
 Now, once a release is created the links from the tag links (qa, testing, production) are not auto-created by the build pipeline. Instead, these has to be created by other means when the operators decide it's right time to deploy a certain version to a certain network. Once decided the link then must be created. This brings us to the `zos-update-worker`
 
 The update worker is a very simple process that watches changes to `tfchain` version as updated by the Council. and apply the correct link.
 
 Say the worker finds out that the zos version on production tfchain is set to `v3.4.5` then it will simply make sure the link from `production` is correctly pointing to the correct release tag. If not, will create that link.
+
+On updates using motions we need to set zos3 and zos4 version and if the new release is safe to upgrade to. For example:
+```json
+{"safe_to_upgrade": true, "version":"v3.12.5", "version_light": "v0.1.1"}`
+```
+- safe_to_upgrade: if the flag is set this indicates that it is safe to update all nodes in the network to this release, if not then the nodes specified in zos-config are the only ones that is getting the upgrade.
+
+For example [here](https://github.com/threefoldtech/zos-config/blob/1a2d1339b219775ca4359535793f067630ed9062/qa.json#L68) if safe_to_upgrade is not set the only nodes getting the new release is nodes of farm 172 other nodes in the network are not.
```

Comment:
don't mention the exact farmid 172 here just say the farms in this list will be upgraded